src/dso_api/settings.py

```diff
@@ -230,6 +230,20 @@
     "ALWAYS_OK": False,
 }
 
+DATAPUNT_ENVIRONMENT = os.getenv("DATAPUNT_ENVIRONMENT")
+if not DATAPUNT_ENVIRONMENT:
+    if schema := os.getenv("SCHEMA_URL") is not None:
+        if "acc." in schema:
+            DATAPUNT_ENVIRONMENT = "acceptance"
+        else:
+            DATAPUNT_ENVIRONMENT = "production"
+    else:
+        DATAPUNT_ENVIRONMENT = "local"
+
 # -- Local app settings
 
-AMSTERDAM_SCHEMA = {"geosearch_disabled_datasets": ["bag", "meetbouten"]}
+AMSTERDAM_SCHEMA = {
+    "geosearch_disabled_datasets": ["bag", "meetbouten"],
+    # For now no acceptance only datasets
+    # "acceptance_only_datasets": []
```

Comment:
willen we dit in de code zo opnemen? Niet in het schema zelf?

Reply:
Je zou het ook in het schema kunnen opnemen , maar ik denk dat je dat soort details, net als of je wel of geen geosearch hebt, niet in het schema wilt hebben